02-Git/Makefile

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
It looks like both these files should be added to the first commit (e.g. squashed), and so this commit is not needed.